airgun/widgets.py

```diff
@@ -2645,7 +2645,25 @@ def read(self):
 
 
 class SearchInput(TextInput):
+    search_field = TextInput(locator=('//input[@aria-label="Search input"]'))
```

Comment:
Could you add some docstring here? I wonder if the docs build fails coz of this.